### PR TITLE
Automatic cleanup!

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,36 +19,37 @@ _[Demo and API Docs](https://elements.polymer-project.org/elements/iron-demo-hel
 
 ##&lt;demo-snippet&gt;
 
-
 `demo-snippet` is a helper element that displays the source of a code snippet and
 its rendered demo. It can be used for both native elements and
 Polymer elements.
 
-    Example of a native element demo
+```html
```
Comment (abdonrd) [inline README.md]: @rictic look this. :)

Comment (rictic) [inline README.md]: Technically it's legal HTML! Plain text is totally fine in HTML.

If we don't want the text as part of these code samples we just need to deindent them in demo-snippet.html

Comment (abdonrd) [inline README.md]: Yes! I know that it's legally, but I think that this text isn't part of the code samples.

Okey! :+1:

Comment (rictic) [inline README.md]: Ah yeah, that may not have been the intent, but according to Markdown it is part of the code, as any text indented by four or more spaces is code.

https://github.com/PolymerElements/iron-demo-helpers/blob/master/demo-snippet.html#L23

Comment (abdonrd) [inline README.md]: Yeah! Thanks @rictic!

**Pull request merged**